docs/users/customization-extendibility.md

```diff
@@ -79,5 +79,14 @@ services:
       - solr:$DDEV_HOSTNAME
 ```
 
+## Providing custom nginx configuration
```

Comment:
I had intended to link to our configuration here but missed it. I have added that link, and provided links to the nginx wiki for additional examples.